bindings_wasm/src/conversations.rs

```diff
@@ -272,10 +272,10 @@ impl Conversations {
   }
 
   #[wasm_bindgen(js_name = createDm)]
-  pub async fn create_dm(&self, account_address: String) -> Result<Conversation, JsError> {
+  pub async fn find_or_create_dm(&self, account_address: String) -> Result<Conversation, JsError> {
```

Comment:
@rygine I thought updating the name in bindings for clarity would be helpful. let me know if there's any concerns